docs/UserGuide.md

```diff
@@ -373,5 +373,5 @@ _Details coming soon ..._
 | **Find**           | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`                                                                                                                       |
 | **Help**           | `help`                                                                                                                                                                           |
 | **Exit**           | `exit`                                                                                                                                                                           |
-| **Stats**          | `stats`                                                                                                                                                                                  |
+| **Stats**          | `stats`                                                                                                                                                                          |
```

Comment:
wait ihni why this was recorded as a change

Reply:
actually can you add in the contact for the find and edit command?

Reply:
Done, I just changed this from draft to ready for review, so take a look.